drivers/saul/init_devs/auto_init_l3gxxxx.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/*
+ * @ingroup     sys_auto_init_saul
+ * @brief       Auto initialization of L3GD20H 3-axis gyroscope sensor
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @file
+ */
+
+#ifdef MODULE_L3GD20H
+
+#include "assert.h"
+#include "log.h"
+#include "saul_reg.h"
+
+#include "l3gd20h.h"
+#include "l3gd20h_params.h"
+
+/**
+ * @brief   Define the number of configured sensors
+ */
+#define L3GD20H_NUM         ARRAY_SIZE(l3gd20h_params)
+
+/**
+ * @brief   Allocate memory for the device descriptors
+ */
+static l3gd20h_t l3gd20h_devs[L3GD20H_NUM];
+
+/**
+ * @brief   Memory for the SAUL registry entries
+ */
+static saul_reg_t saul_entries[L3GD20H_NUM];
+
+/**
+ * @brief   Define the number of saul info
+ */
+#define L3GD20H_INFO_NUM    ARRAY_SIZE(l3gd20h_saul_info)
+
+/**
+ * @brief   Reference the driver struct
+ */
+extern saul_driver_t l3gd20h_saul_driver;
+
+void auto_init_l3gd20h(void)
```

Comment:
```suggestion
void auto_init_l3gxxxx(void)
```

Reply:
Ups, just forgotten when renaming. Unfortunately, it is not enough to recompile it after renaming.